### PR TITLE
Feat/home

### PR DESCRIPTION
**Pull request by renanmav**
### Status

**IN DEVELOPMENT**

- IN DEVELOPMENT: there's active development being done in this branch, don't merge yet

### Todo list

- [x] Tests added and passed
   - [x] `AskHeadline` component
   - [x] Remove `react-test-renderer`

### CODE REVIEW

File: packages/server/src/interface/__tests__/__snapshots__/NodeInterface.spec.ts.snap [hunk 1]
```diff
@@ -2,40 +2,9 @@
...
-        "dummy",
-      ],
-      "title": "Some question",
-      "total_views": 0,
-      "upvotes": 0,
-      "views": 0,
-    },
-  },
+  "errors": Array [
+    [GraphQLError: Cannot query field "title" on type "Question".],
```
Comment (jgcmarins) [inline packages/server/src/interface/__tests__/__snapshots__/NodeInterface.spec.ts.snap]: I presume it's still WIP, but this is a wrong snapshot

File: packages/server/src/modules/question/mutation/__tests__/__snapshots__/EditQuestion.spec.ts.snap [hunk 2]
```diff
@@ -2,29 +2,18 @@
...
-          "fox",
-          "old-meme",
-        ],
-        "title": "Some other question",
-      },
-    },
-  },
+  "errors": Array [
+    [GraphQLError: Field "title" is not defined by type EditQuestionInput.],
+    [GraphQLError: Cannot query field "title" on type "Question".],
```
Comment (jgcmarins) [inline packages/server/src/modules/question/mutation/__tests__/__snapshots__/EditQuestion.spec.ts.snap]: same for those

File: packages/server/src/modules/question/mutation/__tests__/__snapshots__/EditQuestion.spec.ts.snap [hunk 3]
```diff
@@ -2,29 +2,18 @@
...
 Object {
-  "data": Object {
-    "EditQuestion": Object {
-      "error": "You don't own this question",
-      "question": null,
-    },
-  },
+  "errors": Array [
+    [GraphQLError: Field "title" is not defined by type EditQuestionInput.],
+    [GraphQLError: Cannot query field "title" on type "Question".],
```
Comment (jgcmarins) [inline packages/server/src/modules/question/mutation/__tests__/__snapshots__/EditQuestion.spec.ts.snap]: also here

Comment (renanmav) [reply packages/server/src/modules/question/mutation/__tests__/__snapshots__/EditQuestion.spec.ts.snap]: Yeah, I've removed the title field from question model

**Pull request merged**